### ISSUE TITLE
Issue with converted video by mobile ffmpeg on iOS

### ISSUE THREAD

**Issue by j2bmw** [Laerdal/Laerdal.Xamarin.FFmpeg#3]
**Description**
We use ffmpeg wrappers for our .net/c# based Android and iOS apps in Xamarin Forms.  The command line (eg. ffmpeg -i "Front Garden_20210208105006.ps" "Front Garden_20210208105006.mp4") converts the downloaded video to mp4. 

On iOS, the command line is executed successfully. It creates a video file in mp4. However, if the converted mp4 video is played back from the default media player on iOS, the video is barely visible when playing back from iPhone. I can see a little bit at the top, but the rest is blank. However, if I export it to a PC, I can watch the video on iTunes or a file manager app Wondershare Dr.Fone on PC. Please see the attached video file "IMG_0098.MP4" https://user-images.githubusercontent.com/30361626/107766401-825bd100-6d87-11eb-8fc6-777bd726a369.MP4
 and  the conversion log "convert to mp4 ios.txt" [convert to mp4 ios.txt](https://github.com/Laerdal/Laerdal.Xamarin.FFmpeg/files/5971643/convert.to.mp4.ios.txt).

**Expected behavior**
The converted video can be played with the default media player on iOS.

**Current behavior**
The converted mp4 video is played back from the default media player on iOS, the video is barely visible when playing back from iPhone. I can see a little bit at the top, but the rest is blank.

**Screenshots**
No

**Logs**
[convert to mp4 ios.txt](https://github.com/Laerdal/Laerdal.Xamarin.FFmpeg/files/5971643/convert.to.mp4.ios.txt).

**Environment**
- Visual Studio 2019 16.8.4
- Platform: Xamarin Forms 5
 - Architecture: [Android pixel 3 Q]

**Other**

**Comment by framinosona** [comment]
Hi, Your logs mention : 
`2021-02-12 15:57:17.292 tecommobile.iOS[380:68701] WARNING: sws_param option is deprecated and ignored
2021-02-12 15:57:17.295 tecommobile.iOS[380:68701] WARNING: deprecated pixel format used, make sure you did set range correctly`

Did you investigate those? Is the output also the same if you use ffmpeg in your laptop's terminal?

This repo only concerns the port of ffmpeg into Xamarin, for ffmpeg issues (about the tool itself) I'd suggest you post here: https://github.com/tanersener/mobile-ffmpeg

**Comment by j2bmw** [comment]
@framinosona Thanks for looking at the issue. Yes, I have posted the issue at https://github.com/tanersener/mobile-ffmpeg/issues/678. I have been given an expert advice both here and there.

It seems "deprecated pixel format used, make sure you did set range correctly" and "sws_param option is deprecated and ignored" are just warnings, not errors. They can be safely ignored when using ffmpeg from the command-line.

I have resolved a similar issue on Android. Hope I will get this issue under control soon.
